testcases/kernel/controllers/pids/pids.sh

```diff
@@ -196,6 +209,118 @@ case5()
 	fi
 }
 
+case6()
+{
+	tst_res TINFO "set a limit that is smaller than current number of pids"
+	start_pids_tasks2 $max
+
+	lim=$((max - 1))
+	ROD echo $lim \> $testpath/pids.max
+
+	pids_task1 "$testpath/tasks"
+	ret=$?
+
+	if [ "$ret" -eq "2" ]; then
+		tst_res TPASS "fork failed as expected"
+	elif [ "$ret" -eq "0" ]; then
+		tst_res TFAIL "fork didn't fail despite the limit"
+	else
+		tst_res TBROK "pids_task1 failed"
+	fi
+
+	stop_pids_tasks
+}
+
+case7()
+{
+	tst_res TINFO "the number of all child cgroup tasks larger than its parent limit"
+	
+	num=10
+	lim=$((max / num))
+	for i in `seq 1 $num`; do
+		mkdir $testpath/child$i
+		ROD echo $lim \> $testpath/child$i/pids.max
```

Comment:
I guess that we don't even have to limit the children here, all we care about is that the children directories are populated with $max tasks in sum.